app/views/create-client-build.html

```diff
@@ -247,45 +247,60 @@ <h3 class="with-divider">Build Configuration</h3>
-                            <span class="help-block">A name for the android credentials is required.</span>
-                          </div>
+                        <div ng-class="{
+                              'has-error': clientBuildForm.clientCredentialsName.$touched && clientBuildForm.clientCredentialsName.$error.required
+                            }">
+                          <input class="form-control"
+                                  id="clientCredentialsName"
+                                  name="clientCredentialsName"
+                                  type="text"
+                                  ng-model="newClientBuild.clientCredentialsName"
+                                  autocorrect="off"
+                                  autocapitalize="none"
+                                  spellcheck="false"
+                                  required>
+                        </div>
+                        <span class="help-block">A name for the {{newClientBuild.clientType}} credentials.</span>
+                        <div class="has-error" ng-if="clientBuildForm.clientCredentialsName.$touched && clientBuildForm.clientCredentialsName.$error.required">
+                          <span class="help-block">A name for the {{newClientBuild.clientType}} credentials is required.</span>
+                        </div>
 
-                          <label class="required" for="androidKeystore">Android KeyStore</label>
-                          <osc-file-input
-                              id="androidKeystore"
-                              model="newClientBuild.androidKeyStore"
-                              help-text="Password protected PKCS12 file containing a key protected by the same password"
-                              read-as-binary-string="true"
-                              required="true"></osc-file-input>
+                        <label ng-if="newClientBuild.clientType === 'android'" class="required" for="clientCredentials">Android Keystore</label>
+                        <label ng-if="newClientBuild.clientType === 'ios'" class="required" for="clientCredentials">Apple Developer Profile</label>
+                        <osc-file-input
+                            id="clientCredentials"
+                            model="newClientBuild.clientCredentials"
+                            read-as-binary-string="true"
+                            required="true"></osc-file-input>
+                        <div ng-if="newClientBuild.clientType === 'android'" class="help-block"> Password protected PKCS12 file containing a key protected by the same password</div>
+                        <div ng-if="newClientBuild.clientType === 'ios'" class="help-block">
+                          A developer profile file (*.developerprofile) which contains a code signing private key, corresponding developer/distribution certificates, and mobile provisioning profiles. This can be created from XCode.
```

Comment:
Can we add a link to the official apple doc about this subjec: https://help.apple.com/xcode/mac/8.0/#/dev8a2822e0b